docs/components/email/index.html

```diff
@@ -0,0 +1,139 @@
+<p>
+  The {{page.title}} component provides markup to define a
+  single-line text control.
+</p>
+{% endblock %}
+
+{% block content %}
+<section>
+  <header>
+    <h2 id="basic-text-input">Basic Email Input</h2>
+    {# TODO: add section description #}
+  </header>
+
+  <div class="example" id="vue-emailInputDemo" v-cloak>
+    <header>
+      <form class="beta-hxForm">
+        <hx-email-control>
+          <input id="txtLabel" type="text" v-model="label" />
+        </hx-email-control>
+
+        <fieldset>
+          <legend class="beta-hxFieldName">Label Annotation</legend>
+          <hx-checkbox-control>
+            <input id="chkHasAsterisk" type="checkbox" v-model="hasAsterisk" />
+            <label for="chkHasAsterisk">
+              <hx-checkbox></hx-checkbox>
+              Required
+            </label>
+          </hx-checkbox-control>
+
+          <hx-checkbox-control>
+            <input id="chkHasOptional" type="checkbox" v-model="hasOptional" />
+            <label for="chkHasOptional">
+              <hx-checkbox></hx-checkbox>
+              Optional
+            </label>
+          </hx-checkbox-control>
+          <hx-checkbox-control>
+              <input id="chkHasHelpText" type="checkbox" v-model="hasHelpText" />
+              <label for="chkHasHelpText">
+                <hx-checkbox></hx-checkbox>
+                HelpText
+              </label>
+            </hx-checkbox-control>
+            <hx-checkbox-control>
+                <input id="chkHasErrorText" type="checkbox" v-model="hasErrorText" />
+                <label for="chkHasErrorText">
+                  <hx-checkbox></hx-checkbox>
+                  ErrorText
+                </label>
+              </hx-checkbox-control>
+        </fieldset>
+
+        <fieldset>
+          <legend class="beta-hxFieldName">Control Options</legend>
+          <hx-checkbox-control>
+            <input id="chkDisabled" type="checkbox" v-model="isDisabled" />
+            <label for="chkDisabled">
+              <hx-checkbox></hx-checkbox>
+              Disabled
+            </label>
+          </hx-checkbox-control>
+
+          <hx-checkbox-control>
+            <input id="chkRequired" type="checkbox" v-model="isRequired" />
+            <label for="chkRequired">
+              <hx-checkbox></hx-checkbox>
+              Required
+            </label>
+            <p>
+              (Invalid styling applied <i>after</i> user interaction.)
+            </p>
+          </hx-checkbox-control>
+          <hx-checkbox-control :class="{ hxHelpText: hasHelpText}" v-if="helpTextToDisplay" style="display: inline-block;">
+              <label> HelpText</label>
+            <input type="text" v-model="helpText">
+           </hx-checkbox-control>
+           <hx-checkbox-control :class="{hxErrorTextForInput: hasErrorText}" v-if="errorTextToDisplay" style="display: inline-block;">
+              <label> ErrorText</label>
+            <input type="text" v-model="errorText">
```

Comment:
In this block missing the input focus styles .
![image](https://user-images.githubusercontent.com/55139500/77063613-05539a80-6a04-11ea-9185-8b6b7898d8ec.png)

Original:
![image](https://user-images.githubusercontent.com/55139500/77063704-24eac300-6a04-11ea-9f0d-f8150b614420.png)

Reply:
Changes have been done, please review.